src/XRP/model/xrp-path-element.ts

```diff
@@ -43,3 +45,44 @@ export default class XRPPathElement {
     readonly issuer?: string,
   ) {}
 }
+
+/*
+ * A path step in an XRP Ledger Path.
+ * @see: https://xrpl.org/paths.html#path-steps
+ */
+export default class XrpPathElement {
+  /**
+   * Constructs an XrpPathElement from a PathElement.
+   *
+   * @param pathElement a PathElement (protobuf object) whose field values will be used
+   *                    to construct an XrpPathElement
+   * @returns an XrpPathElement with its fields set via the analogous protobuf fields.
+   * @see https://github.com/ripple/rippled/blob/develop/src/ripple/proto/org/xrpl/rpc/v1/transaction.proto#L227
+   */
+  public static from(pathElement: PathElement): XrpPathElement {
+    const account = pathElement.getAccount()?.getAddress()
+    const currency = pathElement.getCurrency()
+    const xrpCurrency = currency && XRPCurrency.from(currency)
```

Comment:
should the idiomatically-cased classes be sure to use idiomatically-cased members?  i.e. this should be `XrpCurrency` ?

Reply:
I suppose we'll catch any of these stragglers when we remove the deprecated code and some things are broken